### PR TITLE
Additional domains

### PR DESCRIPTION
**Pull request by andrewvk**

### CODE REVIEW

File: disposable_email_blacklist.conf [hunk 1]
```diff
@@ -982,6 +1001,7 @@ glucosegrin.com
 gmal.com
 gmatch.org
 gmial.com
+gmx.com
```
Comment (deguif) [inline disposable_email_blacklist.conf]: `gmx.com` is a legitimate email provider. See https://gmx.com

File: disposable_email_blacklist.conf [hunk 2]
```diff
@@ -1374,6 +1404,7 @@ magamail.com
 maggotymeat.ga
 magicbox.ro
 maidlow.info
+mail.mezimages.net
```
Comment (deguif) [inline disposable_email_blacklist.conf]: Please use `mezimages.net` instead of the subdomain.

File: disposable_email_blacklist.conf [hunk 3]
```diff
@@ -1570,6 +1612,7 @@ mountainregionallibrary.net
 mox.pp.ua
 moza.pl
 mr24.co
+msa.minsmail.com
```
Comment (deguif) [inline disposable_email_blacklist.conf]: Please use `minsmail.com` instead of the subdomain

File: disposable_email_blacklist.conf [hunk 4]
```diff
@@ -2624,4 +2691,4 @@ zumpul.com
 zxcv.com
 zxcvbnm.com
 zzi.us
-zzz.com
+zzz.com
```
Comment (deguif) [inline disposable_email_blacklist.conf]: Missing end of line.

File: disposable_email_blacklist.conf [hunk 5]
```diff
@@ -2076,6 +2135,9 @@ spamfree24.info
 spamfree24.net
 spamfree24.org
 spamgoes.in
+spamgourmet.com
```
Comment (deguif) [inline disposable_email_blacklist.conf]: `spamgourment.*` is in the `whitelist.conf` as it requires a registration before the service can be accessible

File: disposable_email_blacklist.conf [hunk 6]
```diff
@@ -2076,6 +2135,9 @@ spamfree24.info
...
+spamgourmet.net
```
Comment (deguif) [inline disposable_email_blacklist.conf]: Same than previous

File: disposable_email_blacklist.conf [hunk 7]
```diff
@@ -2076,6 +2135,9 @@ spamfree24.info
...
+spamgourmet.org
```
Comment (deguif) [inline disposable_email_blacklist.conf]: Same than previous

File: disposable_email_blacklist.conf (hunk 5 above)
Comment (andrewvk) [reply disposable_email_blacklist.conf]: But it allow to create any number of disposable emails.

Comment (andrewvk) [reply disposable_email_blacklist.conf]: Done

**Comment by andrewvk** [comment]
Additional domains come from analysis of user database in numvber of my projects (https://rsdn.org for example).

File: disposable_email_blacklist.conf [hunk 8]
```diff
@@ -1639,6 +1679,7 @@ netricity.nl
 netris.net
 netviewer-france.com
 netzidiot.de
+neverbox.com
```
Comment (deguif) [inline disposable_email_blacklist.conf]: `neverbox.com` is also in the whitelist.

File: disposable_email_blacklist.conf [hunk 9]
```diff
@@ -1834,6 +1883,7 @@ qiq.us
 qisdo.com
 qisoa.com
 qoika.com
+qq.com
```
Comment (deguif) [inline disposable_email_blacklist.conf]: `qq.com` is also in the whitelist.

File: disposable_email_blacklist.conf [hunk 10]
```diff
@@ -2054,6 +2108,8 @@ spambox.info
 spambox.irishspringrealty.com
 spambox.org
 spambox.us
+spamcannon.com
```
Comment (deguif) [inline disposable_email_blacklist.conf]: `spamcannon.com` is also in the whitelist.

File: disposable_email_blacklist.conf [hunk 11]
```diff
@@ -2054,6 +2108,8 @@ spambox.info
...
+spamcannon.net
```
Comment (deguif) [inline disposable_email_blacklist.conf]: Maybe this one should go to `whitelist` too, ping @martenson

File: disposable_email_blacklist.conf [hunk 12]
```diff
@@ -2543,6 +2601,8 @@ xmaily.com
 xn--9kq967o.com
 xost.us
 xoxox.cc
+xoxy.net
```
Comment (deguif) [inline disposable_email_blacklist.conf]: `xoxy.net` is also in the whitelist.

File: disposable_email_blacklist.conf [hunk 13]
```diff
@@ -1978,6 +2031,7 @@ showslow.de
 shrib.com
 shut.name
 shut.ws
+sibmail.com
```
Comment (deguif) [inline disposable_email_blacklist.conf]: `sibmail.com` is also in the whitelist.

File: disposable_email_blacklist.conf [hunk 14]
```diff
@@ -1911,6 +1963,7 @@ s33db0x.com
 sabrestlouis.com
 sackboii.com
 safaat.cf
+safe-mail.net
```
Comment (deguif) [inline disposable_email_blacklist.conf]: `safe-mail.net` is also in the whitelist.

File: disposable_email_blacklist.conf [hunk 15]
```diff
@@ -1867,6 +1917,7 @@ receiveee.com
 recipeforfailure.com
 recode.me
 reconmail.com
+recursor.net
```
Comment (deguif) [inline disposable_email_blacklist.conf]: `recursor.net` is also in the whitelist.

File: disposable_email_blacklist.conf [hunk 16]
```diff
@@ -1680,6 +1722,7 @@ norseforce.com
 nospam.ze.tc
 nospam4.us
 nospamfor.us
+nospammail.net
```
Comment (deguif) [inline disposable_email_blacklist.conf]: `nospammail.net` is also in the whitelist.

File: disposable_email_blacklist.conf [hunk 17]
```diff
@@ -271,6 +276,8 @@ anonymized.org
 anonymousness.com
 ansibleemail.com
 anthony-junkmail.com
+antichef.com
```
Comment (deguif) [inline disposable_email_blacklist.conf]: `antichef.com` is in the whitelist.

File: disposable_email_blacklist.conf [hunk 18]
```diff
@@ -271,6 +276,8 @@ anonymized.org
...
+antichef.net
```
Comment (deguif) [inline disposable_email_blacklist.conf]: `antichef.net` is also in the whitelist

**Comment by andrewvk** [comment]
"Whitelist" name is pretty confusing. OK, I will move specified domains here.

**Comment by andrewvk** [comment]
Is it possible to create to separate lists: one is disposable emails with registration, and one with anonymous disposable emails?

**Comment by deguif** [comment]
Yes, I agree the name `whitelist` is pretty confusing. Ping @martenson

**Comment by deguif** [comment]
Thanks @andrewvk

File: disposable_email_blacklist.conf (hunk 11 above)
Comment (deguif) [reply disposable_email_blacklist.conf]: Sorry @martenson ignore this comment as `spamcannon.net` is already in the whitelist

**Comment by martenson** [comment]
@andrewvk @deguif  e.g. gmail is a disposable email with registration - it is hard to draw a line there...

**Comment by andrewvk** [comment]
Yes, you are right. mail.ru also allows to create disposable synonims. But some sites is intended to use for disposable mails only (e.g. spamgourmet). No one is use such sites for normal mail.

**Comment by deguif** [comment]
@martenson I got your point, but we are talking about disposable domain providers like `https://www.spamgourmet.com` which are stating on their [page](https://www.spamgourmet.com/index.pl?languageCode=EN):

__This spamgourmet email address will be created here the first time BigCorp uses it (you don't have to do anything to create it), and you'll receive at most 3 messages, forwarded to your protected address. The rest will be indelicately consumed.__

There is no comparison possible with what `gmail` and serious mail provider deliver.

**Comment by martenson** [comment]
You make a good case. I am up for separating something like with-registration-but-disposable-oriented from whitelist into another blocklist.

**Pull request merged**